sites/gc-page-feedback/includes/parameters.liquid

```diff
@@ -0,0 +1,57 @@
+{%- comment -%}
+Set parameter variables
```

Comment:
@EricDunsworth Yes there is still a plan to rework how certain plugins work in basic HTML, and this one may be the push necessary to build this new initialization for targeted plugins with Javascript.

So we'll make it in a way that will be compatible with enhanced version (default) + Basic HTML, but not noscript. The basic HTML rework will ensure that the OPUFT will be set with proper values instead of "blank ones".

CC: @duboisp